R/getQuote.R

```diff
@@ -10,7 +10,14 @@ function(Symbols,src='yahoo',what, ...) {
   args <- list(Symbols=Symbols,...)
   if(!missing(what))
       args$what <- what
-  do.call(paste('getQuote',src,sep='.'), args)
+  df <- do.call(paste('getQuote',src,sep='.'), args)
+  if(nrow(df) != length(Symbols)){
+    #merge to generate empty rows for missing results from underlying source
+    df <- merge(list(Symbol = Symbols), df, by = "Symbol", all.x = TRUE)  
```

Comment:
This implicitly converts the list into a data.frame in order to merge it with `df`.  Relying on implicit conversions make it harder to reason about the code. It's better to create a data.frame, as I had in the code deleted in this PR.

Reply:
thanks for the explanation